clientv3/integration/leasing_test.go

```diff
@@ -619,16 +619,29 @@ func TestLeasingTxnOwnerGet(t *testing.T) {
 	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 1})
 	defer clus.Terminate(t)
 
-	lkv, closeLKV, err := leasing.NewKV(clus.Client(0), "pfx/")
+	client := clus.Client(0)
+
+	lkv, closeLKV, err := leasing.NewKV(client, "pfx/")
 	testutil.AssertNil(t, err)
 	defer closeLKV()
 
+	defer func() {
+		// In '--tags cluster_proxy' mode the client need to be closed before
+		// closeLKV(). This interrupts all outstanding watches. Closing by closeLKV()
+		// is not sufficient as (unfortunately) context close does not interrupts Watches.
+		// See ./clientv3/watch.go:
+		// >> Currently, client contexts are overwritten with "valCtx" that never closes. <<
+		clus.TakeClient(0)
```

Comment:
Is this line needed?

Reply:
Yes. Without it the client would be closed 2 times (2nd by clus.Close()), so it would 'fatal/'.